proto/config/vm.proto

```diff
@@ -21,6 +21,12 @@ enum VmMode {
   LEGACY = 5; // HVM, but with fully emulated legacy I/O (IDE disks and e1000 net)
 }
 
+// Boot mechanisms supported by the BIOS of the Virtual Machine
+enum VmBootMode {
+  VMBOOTMODE_LEGACY = 0; // Legacy BIOS, MBR support only
```

Comment:
Maybe I didn't make it clear in the discussions about compatibility and relationship to virtualization mode, but IF we want to be able to set LEGACY with FML (which uses UEFI today), then you can not have a value of zero here (since EVE would not be able to tell the difference  between the controller sending nothing vs the controller sending LEGACY - zero is not encoded on the wire with protobuf).

But note the IF above.

Reply:
Yes Erik, I figured that out. Please, take a look in the updates... I think it should be OK now...